assets/airports/kokc.json

```diff
@@ -0,0 +1,3142 @@
+                "PER": ["PER"],
+                "SGF": ["SGF", "SEGLR"],
+                "TUL": ["TUL"]
+            },
+            "body": ["GULLI"],
+            "rwy": {
+                "KOKC17L": ["#215"],
+                "KOKC17R": ["#215"],
+                "KOKC35L": ["OVAYE", "#175"],
+                "KOKC35R": ["OVAYE", "#175"]
+            },
+            "draw": [
+                ["SGF*", "SEGLR", "GULLI", "OVAYE"],
+                ["TUL*", "GULLI"],
+                ["PER*", "GULLI"]
+            ]
+        },
+        "MURAH3": {
+            "icao": "MURAH3",
+            "name": "Murah Three",
+            "entryPoints": {
+                "AWLIV": [["AWLIV", "A220-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "BELMN": [["BELMN", "A240+"], ["MIICK", "A190+|A280-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "BMART": [["BMART", "A240+"], ["JSTCE", "A190+|A280-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "DRLER": ["DRLER", ["BMART", "A240+"], ["JSTCE", "A190+|A280-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "JTUNE": [["JTUNE", "A240+"], ["FEDRL", "A220-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "LFRED": ["LFRED", ["PAAUL", "A240+"], ["MOMMY", "A190+|A280-|S280"], ["FEDRL", "A220-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "MPYRE": ["MPYRE", ["JTUNE", "A240+"], ["KNEWT", "A190+|A280-|S280"], ["FEDRL", "A220-|S280"], ["MURAH", "A110+|A170-|S280"]],
+                "MURAH": [["MURAH", "A110+|A170-|S280"]],
+                "PAAUL": [["PAAUL", "A240+"], ["FEDRL", "A220-|S280"], ["MURAH", "A110+|A170-|S280"]]
+            },
+            "body": [["JASKA", "A90+|S250"]],
+            "rwy": {
+                "KOKC17L": [["BECTA", "A60+"], ["GOLFS", "A40|S210"]],
+                "KOKC17R": [["BECTA", "A60+"], ["ROHAA", "A40|S210"]],
+                "KOKC35L": [["BOSHY", "A90+"], ["HEDIK", "A80+"], ["MOVIE", "A70+|S220"], ["IMAGE", "A60|S210"], "^HALTO", "#176"],
+                "KOKC35R": [["BOSHY", "A90+"], ["HEDIK", "A80+"], ["MOVIE", "A70+|S220"], ["IMAGE", "A60|S210"], "^HALTO", "#176"]
+            },
+            "draw": [
+                ["LFRED", "PAAUL", "MOMMY", "FEDRL", "MURAH*", "JASKA", "BOSHY", "HEDIK", "MOVIE", "IMAGE"],
+                ["DRLER", "BMART", "JSTCE", "AWLIV", "MURAH", "MIICK", "BELMN"],
+                ["MPYRE", "JTUNE", "KNEWT", "FEDRL"],
+                ["GOLFS", "BECTA", "ROHAA"],
+                ["JASKA", "BECTA"]
+            ]
+        },
+        "RIFFL3": {
+            "icao": "RIFFL3",
+            "name": "Rifle Three",
+            "entryPoints": {
+                "DIMPL": [["DIMPL", "A110+|A170-|S280"], ["RIFFL", "A100+|A150-|S250"]],
+                "DWINE": [["DWINE", "A190+|S280"], ["DIMPL", "A110+|A170-|S280"], ["RIFFL", "A100+|A150-|S250"]],
+                "RIFFL": [["RIFFL", "A100+|A150-|S250"]]
+            },
+            "body": ["FAKEY", ["^AACES", "A60"], "#269"],
```

Comment:
AACES should be at **or above** 6000ft

Reply:
For this and the others, I set it as a mandatory altitude because in the sim, the aircraft will not descend to that altitude. While yes, this is still technically correct, I can verify firsthand that in airline ops, the altitude selector would be set to this “bottom” altitude, despite it being an At Or Above. This is one instance where I need to trick the sim for the purposes of realism. I can change them to At or Above if you insist.

Reply:
All good 👍